### PR TITLE
Superstructure

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

**Review by drorta — DISMISSED** [review]

**Review by hillelv — DISMISSED** [review]
remove bindings before merge

**Comment by drorta** [comment]
Dont merge yet, look what I sent u @YHerm

**Review by maya1414 — DISMISSED** [review]

File: src/main/java/frc/robot/superstructure/Superstructure.java
Comment (drorta) [inline src/main/java/frc/robot/superstructure/Superstructure.java:80]: Doesn't that already exist?

**Review by drorta — DISMISSED** [review]

**Review by maya1414 — DISMISSED** [review]

**Pull request merged**
